src/tilesource.js

```diff
@@ -654,6 +654,7 @@ function processResponse( xhr ){
         } catch (e){
             data = xhr.responseText;
         }
+    /* eslint-disable no-useless-escape */
```

Comment:
Just curious, why the exception here?

Reply:
@iangilman Mostly so you'd look at it! ;)  I trust ESLint's analysis but wanted to double check with you

Reply:
Cool. Looks reasonable, but yeah, regular expressions are already pretty funky!